docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -136,6 +141,11 @@ <h5>
             <td>Symlinks are incorrectly world-writable files after installation from RPM or Deb</td>
             <td>Symlinks are now detected during the building of RPMs and Debs and are copied into the package correctly.</td>
           </tr>
+          <tr>
+             <td>1120</td>
+             <td>Installing fusion without server fails because of a missing vars file</td>
```

Comment:
Since this was introduced and fixed in 5.2.5,  we don't need to document it.

Reply:
Removed it.